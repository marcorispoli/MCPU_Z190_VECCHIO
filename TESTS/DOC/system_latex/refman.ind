\begin{theindex}

  \item {canDriverInterface class Unit Test module}, \hyperpage{7}
  \item {canOpenInterface class Unit Test module}, \hyperpage{7}
  \item {cleanupTestCase}
    \subitem {test\_canDriverInterface}, \hyperpage{19}
  \item {commandProtocol class Unit Test module}, \hyperpage{8}

  \indexspace

  \item {deviceRxSlot}
    \subitem {usingDriver}, \hyperpage{27}
  \item {driverClass}, \hyperpage{15}
    \subitem {driverTxRxData}, \hyperpage{16}
  \item {driverTxRxData}
    \subitem {driverClass}, \hyperpage{16}

  \indexspace

  \item {initTestCase}
    \subitem {test\_canDriverInterface}, \hyperpage{19}

  \indexspace

  \item {rxDone}
    \subitem {usingDriver}, \hyperpage{28}

  \indexspace

  \item {test\_canDriverInterface}, \hyperpage{18}
    \subitem {cleanupTestCase}, \hyperpage{19}
    \subitem {initTestCase}, \hyperpage{19}
    \subitem {test\_canDriverInterface\_1}, \hyperpage{19}
    \subitem {test\_canDriverInterface\_2}, \hyperpage{20}
    \subitem {test\_canDriverInterface\_3}, \hyperpage{20}
    \subitem {test\_canDriverInterface\_4}, \hyperpage{20}
    \subitem {test\_canDriverInterface\_5}, \hyperpage{20}
    \subitem {test\_canDriverInterface\_6}, \hyperpage{20}
    \subitem {test\_canDriverInterface\_7}, \hyperpage{20}
    \subitem {test\_canDriverInterface\_8}, \hyperpage{21}
  \item {test\_canDriverInterface\_1}
    \subitem {test\_canDriverInterface}, \hyperpage{19}
  \item {test\_canDriverInterface\_2}
    \subitem {test\_canDriverInterface}, \hyperpage{20}
  \item {test\_canDriverInterface\_3}
    \subitem {test\_canDriverInterface}, \hyperpage{20}
  \item {test\_canDriverInterface\_4}
    \subitem {test\_canDriverInterface}, \hyperpage{20}
  \item {test\_canDriverInterface\_5}
    \subitem {test\_canDriverInterface}, \hyperpage{20}
  \item {test\_canDriverInterface\_6}
    \subitem {test\_canDriverInterface}, \hyperpage{20}
  \item {test\_canDriverInterface\_7}
    \subitem {test\_canDriverInterface}, \hyperpage{20}
  \item {test\_canDriverInterface\_8}
    \subitem {test\_canDriverInterface}, \hyperpage{21}
  \item {test\_canOpenInterface}, \hyperpage{21}
    \subitem {test\_odRegister\_1}, \hyperpage{22}
    \subitem {test\_odRegister\_2}, \hyperpage{22}
  \item {test\_commandProtocol}, \hyperpage{22}
    \subitem {test\_commandProtocol\_1}, \hyperpage{23}
    \subitem {test\_commandProtocol\_10}, \hyperpage{24}
    \subitem {test\_commandProtocol\_11}, \hyperpage{24}
    \subitem {test\_commandProtocol\_12}, \hyperpage{24}
    \subitem {test\_commandProtocol\_13}, \hyperpage{24}
    \subitem {test\_commandProtocol\_14}, \hyperpage{24}
    \subitem {test\_commandProtocol\_15}, \hyperpage{24}
    \subitem {test\_commandProtocol\_2}, \hyperpage{25}
    \subitem {test\_commandProtocol\_3}, \hyperpage{25}
    \subitem {test\_commandProtocol\_4}, \hyperpage{25}
    \subitem {test\_commandProtocol\_5}, \hyperpage{25}
    \subitem {test\_commandProtocol\_6}, \hyperpage{25}
    \subitem {test\_commandProtocol\_7}, \hyperpage{25}
    \subitem {test\_commandProtocol\_8}, \hyperpage{26}
    \subitem {test\_commandProtocol\_9}, \hyperpage{26}
  \item {test\_commandProtocol\_1}
    \subitem {test\_commandProtocol}, \hyperpage{23}
  \item {test\_commandProtocol\_10}
    \subitem {test\_commandProtocol}, \hyperpage{24}
  \item {test\_commandProtocol\_11}
    \subitem {test\_commandProtocol}, \hyperpage{24}
  \item {test\_commandProtocol\_12}
    \subitem {test\_commandProtocol}, \hyperpage{24}
  \item {test\_commandProtocol\_13}
    \subitem {test\_commandProtocol}, \hyperpage{24}
  \item {test\_commandProtocol\_14}
    \subitem {test\_commandProtocol}, \hyperpage{24}
  \item {test\_commandProtocol\_15}
    \subitem {test\_commandProtocol}, \hyperpage{24}
  \item {test\_commandProtocol\_2}
    \subitem {test\_commandProtocol}, \hyperpage{25}
  \item {test\_commandProtocol\_3}
    \subitem {test\_commandProtocol}, \hyperpage{25}
  \item {test\_commandProtocol\_4}
    \subitem {test\_commandProtocol}, \hyperpage{25}
  \item {test\_commandProtocol\_5}
    \subitem {test\_commandProtocol}, \hyperpage{25}
  \item {test\_commandProtocol\_6}
    \subitem {test\_commandProtocol}, \hyperpage{25}
  \item {test\_commandProtocol\_7}
    \subitem {test\_commandProtocol}, \hyperpage{25}
  \item {test\_commandProtocol\_8}
    \subitem {test\_commandProtocol}, \hyperpage{26}
  \item {test\_commandProtocol\_9}
    \subitem {test\_commandProtocol}, \hyperpage{26}
  \item {test\_odRegister\_1}
    \subitem {test\_canOpenInterface}, \hyperpage{22}
  \item {test\_odRegister\_2}
    \subitem {test\_canOpenInterface}, \hyperpage{22}
  \item {txData}
    \subitem {usingDriver}, \hyperpage{28}
  \item {txToDriver}
    \subitem {usingDriver}, \hyperpage{28}

  \indexspace

  \item {usingDriver}, \hyperpage{26}
    \subitem {deviceRxSlot}, \hyperpage{27}
    \subitem {rxDone}, \hyperpage{28}
    \subitem {txData}, \hyperpage{28}
    \subitem {txToDriver}, \hyperpage{28}

\end{theindex}
