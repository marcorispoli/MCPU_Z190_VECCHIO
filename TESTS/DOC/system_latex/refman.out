\BOOKMARK [0][-]{chapter.1}{\376\377\0001\000\040\000M\000o\000d\000u\000l\000e\000\040\000I\000n\000d\000e\000x}{}% 1
\BOOKMARK [1][-]{section.1.1}{\376\377\0001\000.\0001\000\040\000M\000o\000d\000u\000l\000e\000s}{chapter.1}% 2
\BOOKMARK [0][-]{chapter.2}{\376\377\0002\000\040\000H\000i\000e\000r\000a\000r\000c\000h\000i\000c\000a\000l\000\040\000I\000n\000d\000e\000x}{}% 3
\BOOKMARK [1][-]{section.2.1}{\376\377\0002\000.\0001\000\040\000C\000l\000a\000s\000s\000\040\000H\000i\000e\000r\000a\000r\000c\000h\000y}{chapter.2}% 4
\BOOKMARK [0][-]{chapter.3}{\376\377\0003\000\040\000C\000l\000a\000s\000s\000\040\000I\000n\000d\000e\000x}{}% 5
\BOOKMARK [1][-]{section.3.1}{\376\377\0003\000.\0001\000\040\000C\000l\000a\000s\000s\000\040\000L\000i\000s\000t}{chapter.3}% 6
\BOOKMARK [0][-]{chapter.4}{\376\377\0004\000\040\000M\000o\000d\000u\000l\000e\000\040\000D\000o\000c\000u\000m\000e\000n\000t\000a\000t\000i\000o\000n}{}% 7
\BOOKMARK [1][-]{section.4.1}{\376\377\0004\000.\0001\000\040\000c\000a\000n\000D\000r\000i\000v\000e\000r\000I\000n\000t\000e\000r\000f\000a\000c\000e\000\040\000c\000l\000a\000s\000s\000\040\000U\000n\000i\000t\000\040\000T\000e\000s\000t\000\040\000m\000o\000d\000u\000l\000e}{chapter.4}% 8
\BOOKMARK [2][-]{subsection.4.1.1}{\376\377\0004\000.\0001\000.\0001\000\040\000T\000E\000S\000T\000\040\000D\000E\000S\000C\000R\000I\000P\000T\000I\000O\000N}{section.4.1}% 9
\BOOKMARK [1][-]{section.4.2}{\376\377\0004\000.\0002\000\040\000c\000a\000n\000O\000p\000e\000n\000I\000n\000t\000e\000r\000f\000a\000c\000e\000\040\000c\000l\000a\000s\000s\000\040\000U\000n\000i\000t\000\040\000T\000e\000s\000t\000\040\000m\000o\000d\000u\000l\000e}{chapter.4}% 10
\BOOKMARK [2][-]{subsection.4.2.1}{\376\377\0004\000.\0002\000.\0001\000\040\000T\000E\000S\000T\000\040\000D\000E\000S\000C\000R\000I\000P\000T\000I\000O\000N}{section.4.2}% 11
\BOOKMARK [1][-]{section.4.3}{\376\377\0004\000.\0003\000\040\000c\000o\000m\000m\000a\000n\000d\000P\000r\000o\000t\000o\000c\000o\000l\000\040\000c\000l\000a\000s\000s\000\040\000U\000n\000i\000t\000\040\000T\000e\000s\000t\000\040\000m\000o\000d\000u\000l\000e}{chapter.4}% 12
\BOOKMARK [2][-]{subsection.4.3.1}{\376\377\0004\000.\0003\000.\0001\000\040\000T\000E\000S\000T\000\040\000D\000E\000S\000C\000R\000I\000P\000T\000I\000O\000N}{section.4.3}% 13
\BOOKMARK [3][-]{subsubsection.4.3.1.1}{\376\377\0004\000.\0003\000.\0001\000.\0001\000\040\000I\000N\000V\000A\000L\000I\000D\000-\000F\000R\000A\000M\000E\000\040\000t\000y\000p\000e\000\040\000i\000d\000e\000n\000t\000i\000f\000i\000c\000a\000t\000i\000o\000n\000\040\000t\000e\000s\000t}{subsection.4.3.1}% 14
\BOOKMARK [3][-]{subsubsection.4.3.1.2}{\376\377\0004\000.\0003\000.\0001\000.\0002\000\040\000C\000O\000M\000M\000A\000N\000D\000-\000F\000R\000A\000M\000E\000\040\000t\000y\000p\000e\000\040\000i\000d\000e\000n\000t\000i\000f\000i\000c\000a\000t\000i\000o\000n\000\040\000t\000e\000s\000t}{subsection.4.3.1}% 15
\BOOKMARK [3][-]{subsubsection.4.3.1.3}{\376\377\0004\000.\0003\000.\0001\000.\0003\000\040\000C\000O\000M\000M\000A\000N\000D\000-\000F\000R\000A\000M\000E\000\040\000t\000y\000p\000e\000\040\000a\000n\000d\000\040\000p\000a\000r\000a\000m\000e\000t\000e\000r\000\040\000i\000d\000e\000n\000t\000i\000f\000i\000c\000a\000t\000i\000o\000n\000\040\000t\000e\000s\000t}{subsection.4.3.1}% 16
\BOOKMARK [3][-]{subsubsection.4.3.1.4}{\376\377\0004\000.\0003\000.\0001\000.\0004\000\040\000O\000K\000-\000F\000R\000A\000M\000E\000\040\000t\000y\000p\000e\000\040\000a\000n\000d\000\040\000p\000a\000r\000a\000m\000e\000t\000e\000r\000\040\000i\000d\000e\000n\000t\000i\000f\000i\000c\000a\000t\000i\000o\000n\000\040\000t\000e\000s\000t}{subsection.4.3.1}% 17
\BOOKMARK [3][-]{subsubsection.4.3.1.5}{\376\377\0004\000.\0003\000.\0001\000.\0005\000\040\000D\000E\000L\000A\000Y\000E\000D\000-\000F\000R\000A\000M\000E\000\040\000t\000y\000p\000e\000\040\000a\000n\000d\000\040\000p\000a\000r\000a\000m\000e\000t\000e\000r\000\040\000i\000d\000e\000n\000t\000i\000f\000i\000c\000a\000t\000i\000o\000n\000\040\000t\000e\000s\000t}{subsection.4.3.1}% 18
\BOOKMARK [3][-]{subsubsection.4.3.1.6}{\376\377\0004\000.\0003\000.\0001\000.\0006\000\040\000N\000O\000K\000-\000F\000R\000A\000M\000E\000\040\000t\000y\000p\000e\000\040\000a\000n\000d\000\040\000p\000a\000r\000a\000m\000e\000t\000e\000r\000\040\000i\000d\000e\000n\000t\000i\000f\000i\000c\000a\000t\000i\000o\000n\000\040\000t\000e\000s\000t}{subsection.4.3.1}% 19
\BOOKMARK [3][-]{subsubsection.4.3.1.7}{\376\377\0004\000.\0003\000.\0001\000.\0007\000\040\000A\000C\000K\000-\000F\000R\000A\000M\000E\000\040\000t\000y\000p\000e\000\040\000a\000n\000d\000\040\000p\000a\000r\000a\000m\000e\000t\000e\000r\000\040\000i\000d\000e\000n\000t\000i\000f\000i\000c\000a\000t\000i\000o\000n\000\040\000t\000e\000s\000t}{subsection.4.3.1}% 20
\BOOKMARK [3][-]{subsubsection.4.3.1.8}{\376\377\0004\000.\0003\000.\0001\000.\0008\000\040\000S\000T\000A\000T\000U\000S\000-\000F\000R\000A\000M\000E\000\040\000t\000y\000p\000e\000\040\000a\000n\000d\000\040\000p\000a\000r\000a\000m\000e\000t\000e\000r\000\040\000i\000d\000e\000n\000t\000i\000f\000i\000c\000a\000t\000i\000o\000n\000\040\000t\000e\000s\000t}{subsection.4.3.1}% 21
\BOOKMARK [3][-]{subsubsection.4.3.1.9}{\376\377\0004\000.\0003\000.\0001\000.\0009\000\040\000C\000O\000M\000M\000A\000N\000D\000-\000T\000Y\000P\000E\000\040\000f\000r\000a\000m\000e\000\040\000f\000o\000r\000m\000a\000t\000\040\000c\000r\000e\000a\000t\000i\000o\000n\000\040\000t\000e\000s\000t}{subsection.4.3.1}% 22
\BOOKMARK [3][-]{subsubsection.4.3.1.10}{\376\377\0004\000.\0003\000.\0001\000.\0001\0000\000\040\000D\000E\000L\000A\000Y\000-\000C\000O\000M\000P\000L\000E\000T\000E\000D\000\040\000f\000r\000a\000m\000e\000\040\000f\000o\000r\000m\000a\000t\000\040\000c\000r\000e\000a\000t\000i\000o\000n\000\040\000t\000e\000s\000t}{subsection.4.3.1}% 23
\BOOKMARK [3][-]{subsubsection.4.3.1.11}{\376\377\0004\000.\0003\000.\0001\000.\0001\0001\000\040\000N\000A\000\040\000f\000r\000a\000m\000e\000\040\000f\000o\000r\000m\000a\000t\000\040\000c\000r\000e\000a\000t\000i\000o\000n\000\040\000t\000e\000s\000t}{subsection.4.3.1}% 24
\BOOKMARK [3][-]{subsubsection.4.3.1.12}{\376\377\0004\000.\0003\000.\0001\000.\0001\0002\000\040\000N\000O\000K\000\040\000f\000r\000a\000m\000e\000\040\000f\000o\000r\000m\000a\000t\000\040\000c\000r\000e\000a\000t\000i\000o\000n\000\040\000t\000e\000s\000t}{subsection.4.3.1}% 25
\BOOKMARK [3][-]{subsubsection.4.3.1.13}{\376\377\0004\000.\0003\000.\0001\000.\0001\0003\000\040\000O\000K\000\040\000f\000r\000a\000m\000e\000\040\000f\000o\000r\000m\000a\000t\000\040\000c\000r\000e\000a\000t\000i\000o\000n\000\040\000t\000e\000s\000t}{subsection.4.3.1}% 26
\BOOKMARK [3][-]{subsubsection.4.3.1.14}{\376\377\0004\000.\0003\000.\0001\000.\0001\0004\000\040\000D\000E\000L\000A\000Y\000E\000D\000\040\000f\000r\000a\000m\000e\000\040\000f\000o\000r\000m\000a\000t\000\040\000c\000r\000e\000a\000t\000i\000o\000n\000\040\000t\000e\000s\000t}{subsection.4.3.1}% 27
\BOOKMARK [3][-]{subsubsection.4.3.1.15}{\376\377\0004\000.\0003\000.\0001\000.\0001\0005\000\040\000S\000T\000A\000T\000U\000S\000\040\000f\000r\000a\000m\000e\000\040\000f\000o\000r\000m\000a\000t\000\040\000c\000r\000e\000a\000t\000i\000o\000n\000\040\000t\000e\000s\000t}{subsection.4.3.1}% 28
\BOOKMARK [0][-]{chapter.5}{\376\377\0005\000\040\000C\000l\000a\000s\000s\000\040\000D\000o\000c\000u\000m\000e\000n\000t\000a\000t\000i\000o\000n}{}% 29
\BOOKMARK [1][-]{section.5.1}{\376\377\0005\000.\0001\000\040\000d\000r\000i\000v\000e\000r\000C\000l\000a\000s\000s\000\040\000C\000l\000a\000s\000s\000\040\000R\000e\000f\000e\000r\000e\000n\000c\000e}{chapter.5}% 30
\BOOKMARK [2][-]{subsection.5.1.1}{\376\377\0005\000.\0001\000.\0001\000\040\000D\000e\000t\000a\000i\000l\000e\000d\000\040\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n}{section.5.1}% 31
\BOOKMARK [2][-]{subsection.5.1.2}{\376\377\0005\000.\0001\000.\0002\000\040\000M\000e\000m\000b\000e\000r\000\040\000F\000u\000n\000c\000t\000i\000o\000n\000\040\000D\000o\000c\000u\000m\000e\000n\000t\000a\000t\000i\000o\000n}{section.5.1}% 32
\BOOKMARK [3][-]{subsubsection.5.1.2.1}{\376\377\0005\000.\0001\000.\0002\000.\0001\000\040\000d\000r\000i\000v\000e\000r\000T\000x\000R\000x\000D\000a\000t\000a\000\050\000\051}{subsection.5.1.2}% 33
\BOOKMARK [1][-]{section.5.2}{\376\377\0005\000.\0002\000\040\000t\000e\000s\000t\000\137\000c\000a\000n\000D\000r\000i\000v\000e\000r\000I\000n\000t\000e\000r\000f\000a\000c\000e\000\040\000C\000l\000a\000s\000s\000\040\000R\000e\000f\000e\000r\000e\000n\000c\000e}{chapter.5}% 34
\BOOKMARK [2][-]{subsection.5.2.1}{\376\377\0005\000.\0002\000.\0001\000\040\000D\000e\000t\000a\000i\000l\000e\000d\000\040\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n}{section.5.2}% 35
\BOOKMARK [2][-]{subsection.5.2.2}{\376\377\0005\000.\0002\000.\0002\000\040\000M\000e\000m\000b\000e\000r\000\040\000F\000u\000n\000c\000t\000i\000o\000n\000\040\000D\000o\000c\000u\000m\000e\000n\000t\000a\000t\000i\000o\000n}{section.5.2}% 36
\BOOKMARK [3][-]{subsubsection.5.2.2.1}{\376\377\0005\000.\0002\000.\0002\000.\0001\000\040\000c\000l\000e\000a\000n\000u\000p\000T\000e\000s\000t\000C\000a\000s\000e}{subsection.5.2.2}% 37
\BOOKMARK [3][-]{subsubsection.5.2.2.2}{\376\377\0005\000.\0002\000.\0002\000.\0002\000\040\000i\000n\000i\000t\000T\000e\000s\000t\000C\000a\000s\000e}{subsection.5.2.2}% 38
\BOOKMARK [3][-]{subsubsection.5.2.2.3}{\376\377\0005\000.\0002\000.\0002\000.\0003\000\040\000t\000e\000s\000t\000\137\000c\000a\000n\000D\000r\000i\000v\000e\000r\000I\000n\000t\000e\000r\000f\000a\000c\000e\000\137\0001}{subsection.5.2.2}% 39
\BOOKMARK [3][-]{subsubsection.5.2.2.4}{\376\377\0005\000.\0002\000.\0002\000.\0004\000\040\000t\000e\000s\000t\000\137\000c\000a\000n\000D\000r\000i\000v\000e\000r\000I\000n\000t\000e\000r\000f\000a\000c\000e\000\137\0002}{subsection.5.2.2}% 40
\BOOKMARK [3][-]{subsubsection.5.2.2.5}{\376\377\0005\000.\0002\000.\0002\000.\0005\000\040\000t\000e\000s\000t\000\137\000c\000a\000n\000D\000r\000i\000v\000e\000r\000I\000n\000t\000e\000r\000f\000a\000c\000e\000\137\0003}{subsection.5.2.2}% 41
\BOOKMARK [3][-]{subsubsection.5.2.2.6}{\376\377\0005\000.\0002\000.\0002\000.\0006\000\040\000t\000e\000s\000t\000\137\000c\000a\000n\000D\000r\000i\000v\000e\000r\000I\000n\000t\000e\000r\000f\000a\000c\000e\000\137\0004}{subsection.5.2.2}% 42
\BOOKMARK [3][-]{subsubsection.5.2.2.7}{\376\377\0005\000.\0002\000.\0002\000.\0007\000\040\000t\000e\000s\000t\000\137\000c\000a\000n\000D\000r\000i\000v\000e\000r\000I\000n\000t\000e\000r\000f\000a\000c\000e\000\137\0005}{subsection.5.2.2}% 43
\BOOKMARK [3][-]{subsubsection.5.2.2.8}{\376\377\0005\000.\0002\000.\0002\000.\0008\000\040\000t\000e\000s\000t\000\137\000c\000a\000n\000D\000r\000i\000v\000e\000r\000I\000n\000t\000e\000r\000f\000a\000c\000e\000\137\0006}{subsection.5.2.2}% 44
\BOOKMARK [3][-]{subsubsection.5.2.2.9}{\376\377\0005\000.\0002\000.\0002\000.\0009\000\040\000t\000e\000s\000t\000\137\000c\000a\000n\000D\000r\000i\000v\000e\000r\000I\000n\000t\000e\000r\000f\000a\000c\000e\000\137\0007}{subsection.5.2.2}% 45
\BOOKMARK [3][-]{subsubsection.5.2.2.10}{\376\377\0005\000.\0002\000.\0002\000.\0001\0000\000\040\000t\000e\000s\000t\000\137\000c\000a\000n\000D\000r\000i\000v\000e\000r\000I\000n\000t\000e\000r\000f\000a\000c\000e\000\137\0008}{subsection.5.2.2}% 46
\BOOKMARK [1][-]{section.5.3}{\376\377\0005\000.\0003\000\040\000t\000e\000s\000t\000\137\000c\000a\000n\000O\000p\000e\000n\000I\000n\000t\000e\000r\000f\000a\000c\000e\000\040\000C\000l\000a\000s\000s\000\040\000R\000e\000f\000e\000r\000e\000n\000c\000e}{chapter.5}% 47
\BOOKMARK [2][-]{subsection.5.3.1}{\376\377\0005\000.\0003\000.\0001\000\040\000D\000e\000t\000a\000i\000l\000e\000d\000\040\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n}{section.5.3}% 48
\BOOKMARK [2][-]{subsection.5.3.2}{\376\377\0005\000.\0003\000.\0002\000\040\000M\000e\000m\000b\000e\000r\000\040\000F\000u\000n\000c\000t\000i\000o\000n\000\040\000D\000o\000c\000u\000m\000e\000n\000t\000a\000t\000i\000o\000n}{section.5.3}% 49
\BOOKMARK [3][-]{subsubsection.5.3.2.1}{\376\377\0005\000.\0003\000.\0002\000.\0001\000\040\000t\000e\000s\000t\000\137\000o\000d\000R\000e\000g\000i\000s\000t\000e\000r\000\137\0001}{subsection.5.3.2}% 50
\BOOKMARK [3][-]{subsubsection.5.3.2.2}{\376\377\0005\000.\0003\000.\0002\000.\0002\000\040\000t\000e\000s\000t\000\137\000o\000d\000R\000e\000g\000i\000s\000t\000e\000r\000\137\0002}{subsection.5.3.2}% 51
\BOOKMARK [1][-]{section.5.4}{\376\377\0005\000.\0004\000\040\000t\000e\000s\000t\000\137\000c\000o\000m\000m\000a\000n\000d\000P\000r\000o\000t\000o\000c\000o\000l\000\040\000C\000l\000a\000s\000s\000\040\000R\000e\000f\000e\000r\000e\000n\000c\000e}{chapter.5}% 52
\BOOKMARK [2][-]{subsection.5.4.1}{\376\377\0005\000.\0004\000.\0001\000\040\000D\000e\000t\000a\000i\000l\000e\000d\000\040\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n}{section.5.4}% 53
\BOOKMARK [2][-]{subsection.5.4.2}{\376\377\0005\000.\0004\000.\0002\000\040\000M\000e\000m\000b\000e\000r\000\040\000F\000u\000n\000c\000t\000i\000o\000n\000\040\000D\000o\000c\000u\000m\000e\000n\000t\000a\000t\000i\000o\000n}{section.5.4}% 54
\BOOKMARK [3][-]{subsubsection.5.4.2.1}{\376\377\0005\000.\0004\000.\0002\000.\0001\000\040\000t\000e\000s\000t\000\137\000c\000o\000m\000m\000a\000n\000d\000P\000r\000o\000t\000o\000c\000o\000l\000\137\0001}{subsection.5.4.2}% 55
\BOOKMARK [3][-]{subsubsection.5.4.2.2}{\376\377\0005\000.\0004\000.\0002\000.\0002\000\040\000t\000e\000s\000t\000\137\000c\000o\000m\000m\000a\000n\000d\000P\000r\000o\000t\000o\000c\000o\000l\000\137\0001\0000}{subsection.5.4.2}% 56
\BOOKMARK [3][-]{subsubsection.5.4.2.3}{\376\377\0005\000.\0004\000.\0002\000.\0003\000\040\000t\000e\000s\000t\000\137\000c\000o\000m\000m\000a\000n\000d\000P\000r\000o\000t\000o\000c\000o\000l\000\137\0001\0001}{subsection.5.4.2}% 57
\BOOKMARK [3][-]{subsubsection.5.4.2.4}{\376\377\0005\000.\0004\000.\0002\000.\0004\000\040\000t\000e\000s\000t\000\137\000c\000o\000m\000m\000a\000n\000d\000P\000r\000o\000t\000o\000c\000o\000l\000\137\0001\0002}{subsection.5.4.2}% 58
\BOOKMARK [3][-]{subsubsection.5.4.2.5}{\376\377\0005\000.\0004\000.\0002\000.\0005\000\040\000t\000e\000s\000t\000\137\000c\000o\000m\000m\000a\000n\000d\000P\000r\000o\000t\000o\000c\000o\000l\000\137\0001\0003}{subsection.5.4.2}% 59
\BOOKMARK [3][-]{subsubsection.5.4.2.6}{\376\377\0005\000.\0004\000.\0002\000.\0006\000\040\000t\000e\000s\000t\000\137\000c\000o\000m\000m\000a\000n\000d\000P\000r\000o\000t\000o\000c\000o\000l\000\137\0001\0004}{subsection.5.4.2}% 60
\BOOKMARK [3][-]{subsubsection.5.4.2.7}{\376\377\0005\000.\0004\000.\0002\000.\0007\000\040\000t\000e\000s\000t\000\137\000c\000o\000m\000m\000a\000n\000d\000P\000r\000o\000t\000o\000c\000o\000l\000\137\0001\0005}{subsection.5.4.2}% 61
\BOOKMARK [3][-]{subsubsection.5.4.2.8}{\376\377\0005\000.\0004\000.\0002\000.\0008\000\040\000t\000e\000s\000t\000\137\000c\000o\000m\000m\000a\000n\000d\000P\000r\000o\000t\000o\000c\000o\000l\000\137\0002}{subsection.5.4.2}% 62
\BOOKMARK [3][-]{subsubsection.5.4.2.9}{\376\377\0005\000.\0004\000.\0002\000.\0009\000\040\000t\000e\000s\000t\000\137\000c\000o\000m\000m\000a\000n\000d\000P\000r\000o\000t\000o\000c\000o\000l\000\137\0003}{subsection.5.4.2}% 63
\BOOKMARK [3][-]{subsubsection.5.4.2.10}{\376\377\0005\000.\0004\000.\0002\000.\0001\0000\000\040\000t\000e\000s\000t\000\137\000c\000o\000m\000m\000a\000n\000d\000P\000r\000o\000t\000o\000c\000o\000l\000\137\0004}{subsection.5.4.2}% 64
\BOOKMARK [3][-]{subsubsection.5.4.2.11}{\376\377\0005\000.\0004\000.\0002\000.\0001\0001\000\040\000t\000e\000s\000t\000\137\000c\000o\000m\000m\000a\000n\000d\000P\000r\000o\000t\000o\000c\000o\000l\000\137\0005}{subsection.5.4.2}% 65
\BOOKMARK [3][-]{subsubsection.5.4.2.12}{\376\377\0005\000.\0004\000.\0002\000.\0001\0002\000\040\000t\000e\000s\000t\000\137\000c\000o\000m\000m\000a\000n\000d\000P\000r\000o\000t\000o\000c\000o\000l\000\137\0006}{subsection.5.4.2}% 66
\BOOKMARK [3][-]{subsubsection.5.4.2.13}{\376\377\0005\000.\0004\000.\0002\000.\0001\0003\000\040\000t\000e\000s\000t\000\137\000c\000o\000m\000m\000a\000n\000d\000P\000r\000o\000t\000o\000c\000o\000l\000\137\0007}{subsection.5.4.2}% 67
\BOOKMARK [3][-]{subsubsection.5.4.2.14}{\376\377\0005\000.\0004\000.\0002\000.\0001\0004\000\040\000t\000e\000s\000t\000\137\000c\000o\000m\000m\000a\000n\000d\000P\000r\000o\000t\000o\000c\000o\000l\000\137\0008}{subsection.5.4.2}% 68
\BOOKMARK [3][-]{subsubsection.5.4.2.15}{\376\377\0005\000.\0004\000.\0002\000.\0001\0005\000\040\000t\000e\000s\000t\000\137\000c\000o\000m\000m\000a\000n\000d\000P\000r\000o\000t\000o\000c\000o\000l\000\137\0009}{subsection.5.4.2}% 69
\BOOKMARK [1][-]{section.5.5}{\376\377\0005\000.\0005\000\040\000u\000s\000i\000n\000g\000D\000r\000i\000v\000e\000r\000\040\000C\000l\000a\000s\000s\000\040\000R\000e\000f\000e\000r\000e\000n\000c\000e}{chapter.5}% 70
\BOOKMARK [2][-]{subsection.5.5.1}{\376\377\0005\000.\0005\000.\0001\000\040\000D\000e\000t\000a\000i\000l\000e\000d\000\040\000D\000e\000s\000c\000r\000i\000p\000t\000i\000o\000n}{section.5.5}% 71
\BOOKMARK [2][-]{subsection.5.5.2}{\376\377\0005\000.\0005\000.\0002\000\040\000M\000e\000m\000b\000e\000r\000\040\000F\000u\000n\000c\000t\000i\000o\000n\000\040\000D\000o\000c\000u\000m\000e\000n\000t\000a\000t\000i\000o\000n}{section.5.5}% 72
\BOOKMARK [3][-]{subsubsection.5.5.2.1}{\376\377\0005\000.\0005\000.\0002\000.\0001\000\040\000d\000e\000v\000i\000c\000e\000R\000x\000S\000l\000o\000t}{subsection.5.5.2}% 73
\BOOKMARK [3][-]{subsubsection.5.5.2.2}{\376\377\0005\000.\0005\000.\0002\000.\0002\000\040\000r\000x\000D\000o\000n\000e}{subsection.5.5.2}% 74
\BOOKMARK [3][-]{subsubsection.5.5.2.3}{\376\377\0005\000.\0005\000.\0002\000.\0003\000\040\000t\000x\000D\000a\000t\000a\000\050\000\051}{subsection.5.5.2}% 75
\BOOKMARK [3][-]{subsubsection.5.5.2.4}{\376\377\0005\000.\0005\000.\0002\000.\0004\000\040\000t\000x\000T\000o\000D\000r\000i\000v\000e\000r}{subsection.5.5.2}% 76
\BOOKMARK [0][-]{section*.16}{\376\377\000I\000n\000d\000e\000x}{}% 77
